src/middlewares/one-of.spec.ts

```diff
@@ -151,15 +130,57 @@ describe('error message', () => {
     };
 
     const message = jest.fn(() => 'keep trying');
-    oneOf([check('foo').isInt()], message)(req, {}, () => {
+    oneOf([check('foo').isInt()], { message })(req, {}, () => {
       const context = getOneOfContext(req);
-      expect(context.errors[0]).toHaveProperty('msg', 'keep trying');
+      expect(context.errors).toMatchSnapshot();
       expect(message).toHaveBeenCalledWith({ req });
       done();
     });
   });
 });
 
+describe('should let the user to choose between multiple error types', () => {
+  // TODO: Can't use it.each because it doesn't support done() in TypeScript
```

Comment:
I realise you probably wanted to keep consistency in the file, but since #976, we can use promises instead of callbacks 😉